internal/clients/manager.go

```diff
@@ -60,6 +63,9 @@ type AdminAPIClientsManager struct {
 	// configured.
 	pendingGatewayClients map[string]adminapi.DiscoveredAdminAPI
 
+	// clientFactory is used to create admin API client pointing to the admin API service.
+	clientFactory ClientFactory
```

Comment:
`clientFactory` is not used, please remove it from the struct and its constructor.

Reply:
done.